tests/test_license.py

```diff
@@ -111,6 +112,8 @@ def setUp(self):
             'src/opendr/perception/activity_recognition/datasets/utils/decoder.py',
             'projects/perception/lightweight_open_pose/jetbot/utils/pid.py',
             'src/opendr/perception/compressive_learning/multilinear_compressive_learning/algorithm/trainers.py',
+            'projects/data_generation/synthetic_multi_view_facial_image_generation/test_frontal.py',
+            'projects/data_generation/synthetic_multi_view_facial_image_generation/train.py',
```

Comment:
These files don't exist (at least, not at this location), should this should be removed

Reply:
No, this should not be removed. The paths are corrected.